src/plugins/raven/components/active-edits-table/index.jsx

```diff
@@ -16,84 +16,70 @@ import delegate from './delegate';
 import selector from './selector';
 import TableRow from './TableRow';
 
-class ActiveEditsTable extends React.Component {
-    static propTypes = {
-      accessToken: PropTypes.string.isRequired,
-      collaborationNodes: PropTypes.arrayOf(PropTypes.instanceOf(Message)).isRequired,
-      handleUpdateCollaborations: PropTypes.func.isRequired,
-      title: PropTypes.string,
-    };
+export const ActiveEditsTable = ({
+  accessToken,
+  collaborationNodes,
+  handleUpdateCollaborations,
+  title,
+}) => {
+  const [status, setStatus] = useState(null);
+  const handleRefresh = async () => {
+    setStatus(STATUS_PENDING);
+    await handleUpdateCollaborations(accessToken);
+    setStatus(STATUS_FULFILLED);
+  };
 
-    static defaultProps = {
-      title: 'Active Edits',
-    }
+  useEffect(() => {
```

Comment:
@hunyoboy `useEffect(handleRefresh, []);` is all you need, arrow function isn't needed.

Reply:
@gdbrown i tried that format and it breaks on 'unmount'.

Reply:
@hunyoboy okee dokee, ic, because you're using an async function which returns a value and react is trying to call it.  ok, that's fine